bundles/core/org.eclipse.smarthome.core.thing.xml/src/main/java/org/eclipse/smarthome/core/thing/xml/internal/ThingTypeConverter.java

```diff
@@ -88,12 +93,21 @@ protected ThingTypeConverter(Class clazz, String type) {
     protected ThingTypeXmlResult unmarshalType(HierarchicalStreamReader reader, UnmarshallingContext context,
             Map<String, String> attributes, NodeIterator nodeIterator) throws ConversionException {
 
-        ThingTypeXmlResult thingTypeXmlResult = new ThingTypeXmlResult(new ThingTypeUID(super.getUID(attributes,
-                context)), readSupportedBridgeTypeUIDs(nodeIterator, context), super.readLabel(nodeIterator),
-                super.readDescription(nodeIterator), getChannelTypeReferenceObjects(nodeIterator),
-                getProperties(nodeIterator), super.getConfigDescriptionObjects(nodeIterator));
+        ThingTypeXmlResult thingTypeXmlResult = new ThingTypeXmlResult(
+                new ThingTypeUID(super.getUID(attributes, context)), readSupportedBridgeTypeUIDs(nodeIterator, context),
+                super.readLabel(nodeIterator), super.readDescription(nodeIterator), getListed(attributes),
+                getChannelTypeReferenceObjects(nodeIterator), getProperties(nodeIterator),
+                super.getConfigDescriptionObjects(nodeIterator));
 
         return thingTypeXmlResult;
     }
 
+    protected boolean getListed(Map<String, String> attributes) {
+        String listedFlag = attributes.get("listed");
+        if (listedFlag != null) {
+            return Boolean.parseBoolean(listedFlag);
+        }
+        return true;
```

Comment:
Here you implement default = true, in others contexts you are using default = false